db/schema.rb

```diff
@@ -228,15 +228,15 @@
     t.integer "recipient_id"
     t.string "content_type"
     t.string "attachment_extension"
-    t.string "type"
-    t.datetime "time_discussion_started"
-    t.datetime "time_discussion_completed"
-    t.integer "number_of_prompts"
-    t.datetime "date_extension_assessed"
-    t.boolean "extension_granted"
-    t.integer "assessor_id"
-    t.integer "task_status_id"
-    t.integer "extension_weeks"
+    t.string "type"	
```

Comment:
Revert these changes - it is just white space so no actual change to this file.

Reply:
Also check why this is failing in CI...

Reply:
Hello Andrew, due to i delete something at first, then i make the missing things back to make it as same as the original one.if i revert it, if the file will missing something as before?

Reply:
Or i can drop the worng commit, fix it and repull it